klarna-on-demand/build.gradle

```diff
@@ -1,18 +1,6 @@
 apply plugin: 'com.android.library'
 
 android {
-    packagingOptions {
```

Comment:
Apparently it is not needed after the changes on build.gradle's dependencies section.